PF/Assignment1/LibraryHelperClass.java

```diff
@@ -0,0 +1,152 @@
+package ass1;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class LibraryHelperClass {
+	private Connection connection;
+	private ResultSet resultSet;
+	private Statement statement;
+	private Scanner sc = new Scanner(System.in);
+
+	public LibraryHelperClass() {
+		connection = null;
+		resultSet = null;
+		statement = null;
+		ConnectionUtil conUtil = new ConnectionUtil();
+		connection = conUtil.getConnection();
+	}
+
+	// get connection
+	public Connection getConnection() {
+		return connection;
+	}
+
+	// set connection
+	public void setConnection(Connection connection) {
+		this.connection = connection;
+	}
+
+	public List<Title> getBooksDataRelatedToAuthor() {
+		System.out.println("Enter Name of Author");
+		String query = "SELECT t.title_id,t.title_nm,t.subject_id,t.publisher_id FROM "
+				+ "authors a inner join title_author ta ON a.author_id = ta.author_id "
+				+ "inner join titles t where (t.title_id=ta.title_id AND a.author_nm='"
+				+ sc.nextLine() + "')";
+		List<Title> titleList = new ArrayList<Title>();
+
+		try {
+			statement = connection.createStatement();
+			resultSet = statement.executeQuery(query);
+			while (resultSet.next()) {
+				Title title = new Title(resultSet.getInt(1),
+						resultSet.getString(2), resultSet.getInt(3),
+						resultSet.getInt(4));
+				titleList.add(title);
+			}
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+		return titleList;
+	}
+
+	public boolean addNewMemberAndIssueBook() {
```

Comment:
ACTION: As per requirement, we will be given name of the book and we need to find book has been issued or not. There is no need to add new member and issue book for member. Please check the requirement.